src/c_api/c_api_symbolic.cc

```diff
@@ -658,6 +658,85 @@ int MXSymbolInferShapeEx(SymbolHandle sym,
   API_END();
 }
 
+int MXSymbolInferShapeExInt64(SymbolHandle sym,
```

Comment:
If the internal implementation is the same and only differs by data type, we can create an internal template to do this?